Clarify `execCommand` valid use cases without alternatives. (#40263)
mdn/content: files/en-us/web/api/document/execcommand/index.md

{{ApiRef("DOM")}}{{deprecated_header}}

> [!NOTE]

> > Although the `execCommand` method is deprecated and no longer recommended for use in new projects, there are still some valid use cases that have not yet been replaced, where it remains the only practical option. For example, implementing undo/redo, changing content within the selection range, or applying rich text formatting currently have no alternatives. Developers should use it carefully and ensure compatibility across major browsers.

Reviewer: > "Although the `execCommand` method is ... no longer recommended for use in new projects, there are still some valid use cases ... where it remains the only practical option"

This doesn't make sense to me. Either it shouldn't be used in new projects or there are valid use cases, both can't be true at the same time.

```suggestion
> Although the `execCommand` method is deprecated, there are still some valid use cases that do not yet have viable alternatives, such as implementing undo/redo, changing content within the selection range, or applying rich-text formatting. Developers should use it carefully and test to ensure cross-browser compatibility.
```

Reviewer 2: Both can be true. You can pretend that this feature simply cannot be implemented on the web—for example if the web MIDI API doesn't exist then you simply cannot operate MIDI in a  web app.

Reviewer: > You can pretend that this feature simply cannot be implemented on the web

So there are valid use cases but only in existing projects, new projects should simply give up on trying to interoperate with the browser's undo stack etc? I don't follow this logic.
